csp/impl/types/common_definitions.py

```diff
@@ -62,6 +66,30 @@ def __init__(self, *args, **kwargs):
         ...
 
 
+def _make_pydantic_outputs(kwargs):
+    """Add pydantic functionality to Outputs, if necessary"""
+    from pydantic import create_model
+    from pydantic_core import core_schema
+
+    from csp.impl.wiring.outputs import OutputsContainer
+
+    if None in kwargs:
+        typ = ContainerTypeNormalizer.normalize_type(kwargs[None])
+        model_fields = {"out": (typ, ...)}
+    else:
+        model_fields = {
+            name: (ContainerTypeNormalizer.normalize_type(annotation), ...)
```

Comment:
For my own understanding, why are ellipses used here

Reply:
The pydantic function call `create_model` expects tuples of length 2, for (type, default). If there is no default value, you have to pass it ellipses to indicate that (None would mean a default of None, and it doesn't allow a tuple of length 1).